<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Monitoring</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
    border-image: url(:/hello2.png) 0 0 0 0 stretch stretch;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelCpuTemp">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>55</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 100); font-size: 16px; font-weight: bold; background: transparent;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelGpuTemp">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>87</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 100); font-size: 16px; font-weight: bold; background: transparent;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelSsdTemp">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>122</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 100); font-size: 16px; font-weight: bold; background: transparent;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelUpload">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>175</y>
      <width>120</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 100); font-size: 14px; background: transparent;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDownload">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>196</y>
      <width>120</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 100); font-size: 14px; background: transparent;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelCpuUsage">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>85</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 100); font-size: 22px; font-weight: bold; background: transparent;</string>
    </property>
    <property name="text">
     <string>0%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelGpuUsage">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>85</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 100); font-size: 22px; font-weight: bold; background: transparent;</string>
    </property>
    <property name="text">
     <string>0%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelRamUsage">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>155</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 100); font-size: 22px; font-weight: bold; background: transparent;</string>
    </property>
    <property name="text">
     <string>0%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelSsdUsage">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>155</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 100); font-size: 22px; font-weight: bold; background: transparent;</string>
    </property>
    <property name="text">
     <string>0%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
